Library/Formula/gdub.rb

```diff
@@ -0,0 +1,12 @@
+class Gdub < Formula
+  desc "Gdub is a Gradle wrapper wrapper"
+  homepage "http://gdub.rocks"
+  url "https://github.com/dougborg/gdub/archive/v0.1.0.tar.gz"
+  sha256 "ddf2572cc67b8df3293b1707720c6ef09d6caf73227fa869a73b16239df959c3"
+
+  depends_on "gradle"
+
+  def install
+    bin.install "bin/gw"
+  end
```

Comment:
Please add a test :). See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!